seed/python-sdk/streaming/no-custom-config/README.md

```diff
@@ -62,12 +66,29 @@ will be thrown.
 from seed.core.api_error import ApiError
 
 try:
-    client.dummy.generate(...)
+    client.dummy.generate_stream(...)
 except ApiError as e:
     print(e.status_code)
     print(e.body)
 ```
 
+## Streaming
```

Comment:
We can do this in a follow-up when you introduce the feature to pagination endpoints, but we should make sure to include a `README.md` section for the `with_raw_response` helper. I imagine we might want two sections, where the second is `Access Raw Response (streaming)`.

Something along the lines of [this](https://github.com/fern-api/fern/pull/6689#discussion_r2034144172) for TypeScript.

Reply:
Sounds good!